### DISCUSSION TITLE
unable to create partial chain control: invalid http POST response (nil), method: getblockhash, id: 1, last error=Post "http://localhost:8332": dial tcp [::1]:8332: connectex: No connection could be made because the target machine actively refused it.

### DISCUSSION THREAD

**Discussion by vicalrez** [lightningnetwork/lnd#7458] [Q&A]
unable to create partial chain control: invalid http POST response (nil), method: getblockhash, id: 1, last error=Post "http://localhost:8332": dial tcp [::1]:8332: connectex: No connection could be made because the target machine actively refused it.
Does someone have the same problem?

**Comment by guggero** [comment]
How did you configure the `[bitcoind]` section of your lnd.conf? And how does your bitcoind.conf look like?
